test/config/hosts.yml

```diff
@@ -0,0 +1,18 @@
+toxiproxy:
+  host: 127.0.0.1
```

Comment:
To be consistent with the Toxiproxy vocabulary, I'd use `upstream_host` and `upstream_port`.